backend/src/main/scala/cromwell/backend/validation/FailOnStderrValidation.scala

```diff
@@ -0,0 +1,36 @@
+package cromwell.backend.validation
+
+import wdl4s.types.{WdlBooleanType, WdlStringType}
+import wdl4s.values.{WdlBoolean, WdlString}
+
+import scalaz.Scalaz._
+
+object FailOnStderrValidation {
+  val key = RuntimeAttributesKeys.FailOnStderrKey
+
+  lazy val instance = new FailOnStderrValidation {}
+
+  lazy val default = instance.withDefault(WdlBoolean(false))
+
+  lazy val optional = default.optional
+
+  val missingMessage =
+    s"Expecting $key runtime attribute to be a Boolean or a String with values of 'true' or 'false'"
+}
+
+trait FailOnStderrValidation extends RuntimeAttributesValidation[Boolean] {
```

Comment:
If these are intended to be plugged in for new backend implementations, could you put a little bit (doesn't need to be nearly as robust as the other comments I asked for) on how to use them?